consensus/service/src/config/tokens.rs

```diff
@@ -5,12 +5,74 @@
 use crate::consensus_service::ConsensusServiceError;
 use mc_common::HashSet;
 use mc_consensus_enclave::FeeMap;
+use mc_crypto_keys::{DistinguishedEncoding, Ed25519Public};
+use mc_crypto_multisig::SignerSet;
 use mc_transaction_core::{tokens::Mob, Token, TokenId};
-use serde::{Deserialize, Serialize};
+use serde::{Deserialize, Deserializer, Serialize, Serializer};
 use std::{fs, iter::FromIterator, path::Path};
 
+mod der_signer_set {
+    use super::*;
+
+    /// A helper struct for ser/derserializing a SignerSet that uses hex-encoded
+    /// DER representation of Ed25519 public keys.
+    #[derive(Serialize, Deserialize)]
+    struct DerSignerSet {
+        signers: Vec<String>,
```

Comment:
DER is the binary format, and that gets Base64 encoded for PEM, and elsewhere I've used `Vec<Vec<u8>>` for this...

Reply:
c.f. https://docs.rs/pem/latest/pem/fn.parse_many.html

Reply:
Done